ros_ign_gazebo/launch/ign_gazebo.launch.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Launch Gazebo server and client with command line arguments."""
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import IncludeLaunchDescription
+from launch.conditions import IfCondition
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+from launch.substitutions import ThisLaunchFileDir
+
+
+def generate_launch_description():
+
+    return LaunchDescription([
+        DeclareLaunchArgument('gui', default_value='true',
+                              description='Set to "false" to run headless.'),
+
+        DeclareLaunchArgument('server', default_value='true',
+                              description='Set to "false" not to run gzserver.'),
```

Comment:
I'm on the fence about splitting this into `gui` and `server`. Ignition Gazebo already supports this internally with the `-s` and `-g` flags. I feel like we could be duplicating the efforts here. I don't feel too strongly about it, but I have a feeling that `ign-gazebo` should know better how to manage the lifecycle for the GUI and server processes.

Reply:
Another argument for a single entry point is that now we have arguments split between gui and server args. Users of `ign gazebo` don't need to think about these differences, so I think users of these launch files also shouldn't have to. Another downside is that arguments that apply to both, such as `-v 4`, would need to be duplicate.

The more I think the more I'm convinced that we should let `ign gazebo` handle the GUI and server processes.

What do you think?